speech/src/main/java/com/example/speech/InfiniteStreamRecognize.java

```diff
@@ -63,19 +60,35 @@ public class InfiniteStreamRecognize {
   private static boolean newStream = true;
   private static double bridgingOffset = 0;
   private static boolean lastTranscriptWasFinal = false;
+  private static boolean stopRecognition = false;
   private static StreamController referenceToStreamController;
   private static ByteString tempByteString;
 
-  public static void main(String... args) {
+  public static void main(String... args) throws LineUnavailableException {
     InfiniteStreamRecognizeOptions options = InfiniteStreamRecognizeOptions.fromFlags(args);
     if (options == null) {
       // Could not parse.
       System.out.println("Failed to parse options.");
       System.exit(1);
     }
 
+    // TODO(developer): Replace the variables before running the sample or use default
+    // the number of samples per second
+    int sampleRate = 16000;
+    // the number of bits in each sample
+    int sampleSizeInBits = 16;
+    // the number of channels (1 for mono, 2 for stereo, and so on)
+    int channels = 1;
+    // indicates whether the data is signed or unsigned
+    boolean signed = true;
+    // indicates whether the data for a single sample is stored in big-endian byte
+    // order (false means little-endian)
+    boolean bigEndian = false;
+
+    MicBuffer micBuffer = new MicBuffer(sampleRate, sampleSizeInBits, channels, signed, bigEndian);
     try {
-      infiniteStreamingRecognize(options.langCode);
+      infiniteStreamingRecognize(options.langCode, micBuffer, sampleRate,
+              RecognitionConfig.AudioEncoding.LINEAR16);
```

Comment:
I understand that this method exits when a stream has a word "exit".
It would be useful to document it in the code. Consider renaming variable to be more verabit (eg "exitRecognized") or write a comment.
nit: Otherwise, if a user ends up killing the app, a graceful termination on SIGKILL might be useful if resources require release.

Reply:
I added a comment on how to stop, and this will also be printed when the application runs.

Reply:
We have guidelines how code sample should look like. This repository is intended to host code used mainly for documentation code snippets. See go/code-snippets-101#what-is-a-code-snippet for the definition. It is not intended for demo applications or other complex solutions.
I agree that _printing_ it makes more sense than place a comment. However, I see no code line that prints this instruction.

Reply:
Could you please send a direct link, or if this link needs google account, could you please copy it here a phrase